api/src/functions/query-requirements/glpk-js-fixed.d.ts

```diff
@@ -0,0 +1,87 @@
+// Taken from: https://github.com/jvail/glpk.js/blob/3d2ff55364fd61689f027de9abda179f4e09c7e9/dist/glpk.d.ts
```

Comment:
The default GLPK `.d.ts` file bundled with this version of `glpk.js` has issues. Fixes have been applied to the package but it has not been re-deployed with these fixes. This solves the problem for now